spec/index.html

```diff
@@ -720,10 +720,20 @@ <h3>Properties of simple entailment and satisfiability</h3>
 
 	<p class="fact"> The set F of facts in an interpretation I is F(I) = {&nbsp;RE(x, y, z)&#65372;&lt;x, z&gt; is in IEXT(y)&nbsp;}. The set of facts is the set of propositions which are true in the interpretation. </p>
 
+	<p>We define the <dfn>set of reifications</dfn> in an interpretation as follows:</p>
+
+	<p class="fact"> The set R of reifications in an interpretation I is R(I) = {&nbsp;RE(x, y, z)&#65372;
+		x is in IR,
+		y is rdf:reifies,
+		z is a triple term and 
+		&lt;x, z&gt; is in IEXT(y)&nbsp;}. 
```

Comment:
Starting with @doerthe's [re-draft of this sentence](https://github.com/w3c/rdf-semantics/pull/144/files#r2291118507), I've made some more small tweaks (I added a missing terminal comma; added an `nbsp`; and deleted a couple of spaces from EOLs)...
```suggestion
	<p class="fact"> The set R of reifications in an interpretation I is R(I) = {&nbsp;RE(x, y, z)&nbsp;&#65372;
		x is in IR,
		y is IS(rdf:reifies),
		z is in IPR(I), and
		&lt;x, z&gt; is in IEXT(y)&nbsp;}.
```